app/assets/src/components/common/DetailsSidebar/SampleDetailsMode/sample_details_mode.scss

```diff
@@ -66,6 +91,7 @@
 
   .emptyValue {
     color: $light-grey;
+    font-weight: 400;
```

Comment:
Same as before... Even if not a full font, you should use `$font-weight-regular`